api/api.proto

```diff
@@ -2542,7 +2544,7 @@ message SdkVersion {
     // SDK version major value of this specification
     Major = 0;
     // SDK version minor value of this specification
-    Minor = 23;
+    Minor = 24;
```

Comment:
Thanks, this looks good. I can see that StorageResource is used by StorageNode. Could you please add a comment to the SDK_CHANGELOG.md at the root of the file.

Reply:
yes done and revised.